### PR TITLE
e2e node selector and resources in one NUMA zone

### PR DESCRIPTION
**Pull request by jlojosnegros**
Having two different nodes which fulfill the pod node selector
check the pod is scheduled in the node having enough resources
in one NUMA zone

### CODE REVIEW

**Comment by jlojosnegros** [comment]
added test_id

File: test/e2e/serial/workload_placement_test.go [hunk 1]
```diff
@@ -444,6 +444,127 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
...
+			pod.Spec.SchedulerName = schedulerName
+			pod.Spec.Containers[0].Resources.Limits = requiredRes
+			pod.Spec.NodeSelector = map[string]string{
+				labelName: labelValue,
+			}
+
+			err = fxt.Client.Create(context.TODO(), pod)
+			Expect(err).NotTo(HaveOccurred(), "unable to create pod %q", pod.Name)
+
+			By("waiting for node to be up&running")
```
Comment (fromanirh) [inline test/e2e/serial/workload_placement_test.go:554]: typo: for the pod

**Review by fromanirh — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**